src/User.php

```diff
@@ -2746,6 +2746,18 @@ public function showForm($ID, array $options = [])
                     'right'  => 'all'
                 ]);
                 echo "</td></tr>";
+
+                echo "<tr class='tab_bg_1'>";
+                echo "<td><label for='dropdown_profiles_id$grouprand'>" .  __('System user') . "</label></td><td>";
+                Html::showCheckbox([
```

Comment:
I am not sure it is a good idea to permit to edit this field manually. IMHO, system users are meant to be used/updated only by system.

Reply:
I agree with cedric-anne